pipeline/metadata/5.1.yaml

```diff
@@ -0,0 +1,480 @@
+- name: "test-bootstrap-custom-ssl-dashboard-port-and-apply-spec"
+  suite: "suites/pacific/cephadm/tier-1_ssl_dashboard_port.yaml"
+  global-conf: "conf/pacific/cephadm/tier-1_3node_cephadm_bootstrap.yaml"
+  platform: "rhel-8"
+  rhbuild: "5.1"
```

Comment:
We know the RHCS version as it is referencing the file... do we want to have `rhbuild`?

Reply:
We are letting the user pass a metadata file location to be used if he has created a custom one in cephci. This would make sure that we're not asking for the file name to be the build name specifically.

Also Its easier to write the logic for building CLI. Otherwise we would need to specifically add the rhbuild based on the file name for the CLI. 

@psathyan Please let us know your thoughts.

Reply:
Thanks @manasagowri

Glad to hear we have that support... yes, I agree it makes sense to have `rhbuild` to support the scenario of custom metadata file.